elasticapm/traces.py

```diff
@@ -274,17 +285,22 @@ def begin_span(
             start=start,
         )
 
-    def end_span(self, skip_frames=0, duration=None):
+    def end_span(self, skip_frames=0, duration=None, outcome=None):
         """
         End the currently active span
         :param skip_frames: numbers of frames to skip in the stack trace
         :param duration: override duration, mostly useful for testing
+        :param outcome: outcome of the span, either success, failure or unknown
         :return: the ended span
         """
         span = execution_context.get_span()
         if span is None:
             raise LookupError()
 
+        # only overwrite span outcome if it is still unknown
+        if span.outcome == "unknown":
```

Comment:
```suggestion
        if outcome and span.outcome == "unknown":
```

I assume that the outcome shouldn't be `None` -- if I'm incorrect then you can ignore this suggestion.

Reply:
I changed the default to "unknown" in 37ea77d, that should take care of this, right?

Reply:
Yep!